assets/tailwind.config.js

```diff
@@ -29,8 +29,18 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        ...colors
-      }
+        ...colors,
+        "gray-lighter": "#b0b5c0",
+        "gray-light": "#788093",
+        "navy": {
+          500: "#104c8f"
+	},
+        "logan-express-bb": "#f16823",
```

Comment:
I'm open to suggestions about how to name these colors. I do get the sense that the jury's still out regarding how MBTA-brand-specific colors should be named, but it seems to me like there's a place for some very specific hard-coded brand colors, like "The Logan Express Braintree color".

Aaaaand actually, I'm kind of going back to my original thought that the MBTA blue line blue (or subway blue) or whatever should also get its own very explicit and obvious name (`mbta-subway-blue`, for instance), rather than relying on `blue-500` being the subway-brand color.